pipelines/generate_test_data.R

```diff
@@ -203,10 +203,7 @@ generate_fake_state_level_data <-
 
     # Write out-of-sample state-level data to comparison directory
     state_data |>
-      filter(reference_date > end_reference) |>
-      write_parquet(path(comp_dir, "latest_comprehensive",
-        ext = "parquet"
-      ))
```

Comment:
This is the only non-style change. I no longer `filter(reference_date > end_reference)`.